dev_guide/deployments.adoc

```diff
@@ -191,6 +191,53 @@ To see what the rollback would look like without performing the rollback:
 $ oc rollback <deployment_config> --dry-run
 ----
 
+[[executing-commands-inside-a-container-deployments]]
+== Executing Commands Inside a Container
```

Comment:
This is the first I've seen the title be appended at the end. I usually prepend it, e.g.:

```
[[deployments-executing-commands-inside-a-container]]
```